tests/components/test_nlp.py

```diff
@@ -68,6 +69,38 @@ def test_process(kwik_e_mart_nlp):
     }
 
 
+def test_query_info_contains_language_information(kwik_e_mart_nlp):
+    file_to_test = None
+    for file in kwik_e_mart_nlp.resource_loader.file_to_query_info.keys():
+        if 'train.txt' in file:
+            file_to_test = file
+            break
+    query = kwik_e_mart_nlp.resource_loader.file_to_query_info[file_to_test]['queries'][0].query
+    assert query.language == 'en'
+    assert query.locale == 'en_CA'
+
+
+def test_process_contains_language_information(kwik_e_mart_nlp):
+    # Timestamp is set to Jan 2nd 2020 so that the resolve thanksgiving date will be in 2020
+    result = kwik_e_mart_nlp.process('Is the Main Street location open for Thanksgiving?',
+                                     timestamp=1578025558)
+    for entity in result['entities']:
+        if entity['text'] == 'Thanksgiving':
```

Comment:
will this test fail next year (2021)?

Reply:
No it wont since I have set the timestamp to the year 2020 (line 86)